// kaokun/lib/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.random = exports.silly = exports.bored = exports.cute = exports.happy = exports.surprised = exports.love = exports.angry = exports.hurt = exports.sad = exports.fun = exports.greeting = exports.kaomoji = void 0;
const libs_1 = require("./libs/libs");
const kaomoji = ({ seed, emotion, maxLength, sides, matchingEyes } = {}) => (0, libs_1.getKao)({ seed, emotion, maxLength, sides, matchingEyes });
exports.kaomoji = kaomoji;
const greeting = (seed, maxLength) => (0, libs_1.getKao)({ seed, emotion: 'greeting', maxLength });
exports.greeting = greeting;
const fun = (seed, maxLength) => (0, libs_1.getKao)({ seed, emotion: 'fun', maxLength });
exports.fun = fun;
const sad = (seed, maxLength) => (0, libs_1.getKao)({ seed, emotion: 'sad', maxLength });
exports.sad = sad;
const hurt = (seed, maxLength) => (0, libs_1.getKao)({ seed, emotion: 'hurt', maxLength });
exports.hurt = hurt;
const angry = (seed, maxLength) => (0, libs_1.getKao)({ seed, emotion: 'angry', maxLength });
exports.angry = angry;
const love = (seed, maxLength) => (0, libs_1.getKao)({ seed, emotion: 'love', maxLength });
exports.love = love;
const surprised = (seed, maxLength) => (0, libs_1.getKao)({ seed, emotion: 'surprised', maxLength });
exports.surprised = surprised;
const happy = (seed, maxLength) => (0, libs_1.getKao)({ seed, emotion: 'happy', maxLength });
exports.happy = happy;
const cute = (seed, maxLength) => (0, libs_1.getKao)({ seed, emotion: 'cute', maxLength });
exports.cute = cute;
const bored = (seed, maxLength) => (0, libs_1.getKao)({ seed, emotion: 'bored', maxLength });
exports.bored = bored;
const silly = (seed, maxLength) => (0, libs_1.getKao)({ seed, emotion: 'silly', maxLength });
exports.silly = silly;
const random = (maxLength) => (0, libs_1.getKao)({ maxLength });
exports.random = random;
